mkdocs_swagger_ui_tag/plugin.py

```diff
@@ -107,94 +120,104 @@ def on_post_page(self, output, page, config, **kwargs):
         Add javascript code to update iframe height
         Create a html with Swagger UI for iframe
         """
+        # Using filename filter for performance
+        # https://github.com/blueswen/mkdocs-swagger-ui-tag/issues/25
+        if filter_list := self.config["filter_filenames"]:
+            if page.file.name not in filter_list:
+                return output
 
         soup = BeautifulSoup(output, "html.parser")
         swagger_ui_list = soup.find_all("swagger-ui")
         iframe_id_list = []
         grouped_list = []
 
-        if len(swagger_ui_list) > 0:
-            css_dir = utils.get_relative_url(
-                utils.normalize_url("assets/stylesheets/"), page.url
-            )
-            js_dir = utils.get_relative_url(
-                utils.normalize_url("assets/javascripts/"), page.url
+        # No tags found, we can return earlier
+        if len(swagger_ui_list) == 0:
+            return output
```

Comment:
The whole block down this change is just un-indented.
The diff is a bit weird on that, sorry.